build-system/common/ctrlcHandler.js

```diff
@@ -17,7 +17,7 @@
 const colors = require('ansi-colors');
 const log = require('fancy-log');
 const {execScriptAsync, exec} = require('./exec');
-const {isTravisBuild} = require('./travis');
+const {isCiBuild} = require('./ci');
```

Comment:
Nit: CI is a very familiar acronym to us, but out of context it may not be obvious what a module named `ci` is for. (especially when looking at the full list of files under `build-system/common`). `ci_env` or `circle_ci` or even `build_env` are a few suggestions that come to mind when thinking about what this module might encompass

Reply:
To add a contrary opinion, I think `build-system/common/ci` is clear enough - but I don't feel strongly about this at all

Reply:
The idea is for this file to contain a set of functions applicable to continuous integration in general, no matter which provider is being used. In future, it's likely to include CI-specific functions that encompass more than just environment vars.

Also, the term CI is fairly standard in OSS, with its own [wiki page](https://en.wikipedia.org/wiki/Continuous_integration). I went with `ci.js` because `continuous_integration.js` was cumbersome to type :)